runtime/src/main/resources/application-zitadel.properties

```diff
@@ -3,28 +3,13 @@
 # SPDX-FileCopyrightText: Huawei Inc.
 #
 #
-# spring security oauth2 config
-logging.level.org.springframework.web=info
-logging.level.org.springframework.security=debug
-# set authorization.token.type: JWT or OpaqueToken
-authorization.token.type=OpaqueToken
-# set authorization server endpoint and client configs
-authorization.server.endpoint=http://localhost:8081
-authorization.api.client.id=
-authorization.api.client.secret=
-authorization.swagger.ui.client.id=
-# spring security oauth2 config when using token type JWT
-spring.security.oauth2.resourceserver.jwt.issuer-uri=${authorization.server.endpoint}
-spring.security.oauth2.resourceserver.jwt.jwk-set-uri=${authorization.server.endpoint}/oauth/v2/keys
-# spring security oauth2 config when using token type OpaqueToken
-spring.security.oauth2.resourceserver.opaquetoken.introspection-uri=${authorization.server.endpoint}/oauth/v2/introspect
-spring.security.oauth2.resourceserver.opaquetoken.client-id=${authorization.api.client.id}
-spring.security.oauth2.resourceserver.opaquetoken.client-secret=${authorization.api.client.secret}
-# springdoc openapi security oauth2 config
-springdoc.show-login-endpoint=true
-springdoc.swagger-ui.oauth.use-pkce-with-authorization-code-grant=true
-springdoc.swagger-ui.oauth.clientId=${authorization.swagger.ui.client.id}
-oauth.authorization.url=${authorization.server.endpoint}/oauth/v2/authorize
-oauth.token.url=${authorization.server.endpoint}/oauth/v2/token
-oauth.protected.api.client.id=
-oauth.protected.api.client.secret=
\ No newline at end of file
+
+# set authorization server endpoint
```

Comment:
this can be a bit misleading. 
We can either delete this file and keep only zitadel-testbed
or we can keep this file with empty values.

Reply:
We no longer have a zitadel-testbed configuration file. I have placed all zitadel configuration items in the application-zitadel.properties configuration file.